dephell/commands/deps_convert.py

```diff
@@ -61,6 +61,18 @@ def __call__(self) -> bool:
                 return False
             self.logger.debug('resolved')
 
+        # apply envs if needed
+        if 'envs' in self.config:
+            if not should_be_resolved:
+                self.logger.debug('resolving...')
+                resolved = resolver.resolve(level=1, silent=self.config['silent'])
+                if not resolved:
+                    conflict = analyze_conflict(resolver=resolver)
+                    self.logger.warning('conflict was found')
+                    print(conflict)
```

Comment:
print?

Reply:
>.<

Reply:
Yeah, that's right. This is the printing of conflict info.

Reply:
Why don't use logger instead?

Reply:
Because this is really long output.

Reply:
Please, open an issue if this is producing some troubles for you. I will be happy to fix it. This is not related to this PR because conflict info is printed in the same way for all commands.